### PR TITLE
Sort unsorted.m3u

### PR DESCRIPTION
**Pull request by freearhey**
- all links were moved to the corresponding files
- created a new file `us_simultv.m3u` for links from [SimulTV](https://simultv.com/) service

### CODE REVIEW

File: streams/al.m3u [hunk 1]
```diff
@@ -63,3 +63,27 @@ http://monamour.agency/ntv/livestream/index.m3u8
 http://monamour.agency/johaniter007web/index.m3u8
 #EXTINF:-1 tvg-id="" status="blocked",Shqiponja (720p) [Not 24/7]
 http://ott.iptvshqipott.com:8080/live/shqiponja/tv/236.m3u8
+#EXTINF:-1 tvg-id="" status="online",Korça + (1080p) [Not 24/7]
+http://32.shqiptv.org/korca/albplus/index.m3u8
+#EXTINF:-1 tvg-id="" status="online",Neser TV
+https://nesertv.live/ntv/livestream/tracks-v1a1/index.m3u8
```
Comment (LaneSh4d0w) [inline streams/al.m3u:69]:
```suggestion
http://nesertv.live/ntv/livestream/index.m3u8
```
cert issue

File: streams/al.m3u [hunk 2]
```diff
@@ -63,3 +63,27 @@ http://monamour.agency/ntv/livestream/index.m3u8
...
+#EXTINF:-1 tvg-id="" status="online",AlbKanale Music TV
+https://albportal.net/albkanalemusic.m3u8
+#EXTINF:-1 tvg-id="ReportTV.al" status="online",Report TV
+https://deb10stream.duckdns.org/hls/stream.m3u8
+#EXTINF:-1 tvg-id="" status="timeout",ALB Music (720p) [Not 24/7]
+http://albmusic.dyndns.tv:1935/albuk/albmus.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",News 24 Albania
+http://tv.balkanweb.com:8081/news24/livestream/playlist.m3u8
+#EXTINF:-1 tvg-id="KlanPlus.al" status="timeout",Klan Plus
+http://93.157.62.180/KlanPlus/index.m3u8
```
Comment (LaneSh4d0w) [inline streams/al.m3u:85]:
```suggestion
http://199.127.60.117:8081/live/klanplus/playlist.m3u8
```

File: streams/ar.m3u [hunk 3]
```diff
@@ -282,3 +282,7 @@ https://stmv1.voxhdnet.com/aniplus/aniplus/playlist.m3u8
 http://190.61.41.182:8000/play/a2rn/index.m3u8
 #EXTINF:-1 tvg-id="" status="timeout",ElOnce (1080p) [Not 24/7]
 https://elonceovh.elonce.com/hls/live.m3u8
+#EXTINF:-1 tvg-id="Canal26.ar" status="online",Canal 26 (720p)
+http://live-edge01.telecentro.net.ar:1935/live/26hd-720/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/ar.m3u:286]:
```suggestion
https://live-edge01.telecentro.net.ar/live/smil:c26.smil/playlist.m3u8
```
adaptive

File: streams/ar.m3u [hunk 4]
```diff
@@ -282,3 +282,7 @@ https://stmv1.voxhdnet.com/aniplus/aniplus/playlist.m3u8
...
+#EXTINF:-1 tvg-id="NETTV.ar" status="online",NET TV (720p)
+https://unlimited1-us.dps.live/nettv/nettv.smil/nettv/livestream1/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/ar.m3u:288]:
```suggestion
https://unlimited1-us.dps.live/nettv/nettv.smil/playlist.m3u8
```
adaptive

File: streams/ba.m3u [hunk 5]
```diff
@@ -25,3 +25,9 @@ http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/watch?v
 http://webtvstream.bhtelecom.ba/malta.m3u8
 #EXTINF:-1 tvg-id="" status="blocked",РТРС (576p) [Geo-blocked]
 https://parh.rtrs.tv/tv/live/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",Super TV Media
+https://mirtv.club/live/mirtv/index.m3u8
+#EXTINF:-1 tvg-id="" status="online",Glas Drine
+http://glasdrine.cutuk.net:8081/433ssdsw/GlasDrineSD/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="error",RTV Herceg-Bosne
+https://prd-hometv-live-open.spectar.tv/ERO_1_083/576p/chunklist.m3u8
```
Comment (LaneSh4d0w) [inline streams/ba.m3u:33]:
```suggestion
https://prd-hometv-live-open.spectar.tv/ERO_1_083/playlist.m3u8
```
adaptive + not 24/7

File: streams/br.m3u [hunk 6]
```diff
@@ -235,3 +235,27 @@ https://evpp.mm.uol.com.br/geob_band/bandnewstv/playlist.m3u8
 https://playplusgoya-lh.akamaihd.net/i/pp_gna@377833/index_720_av-p.m3u8
 #EXTINF:-1 tvg-id="" status="timeout",BM&C [Not 24/7]
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCoja0MNI4SIvXqJR3szbSfw/live
+#EXTINF:-1 tvg-id="RedeTV.br" status="online",RedeTV! Tocantins (720p) [Not 24/7]
+https://59f1cbe63db89.streamlock.net:1443/redetvro/_definst_/redetvro/playlist.m3u8
+#EXTINF:-1 tvg-id="TVEBahia.br" status="online",TVE RS (1080p)
+http://streaming.procergs.com.br:1935/tve/stve/playlist.m3u8
+#EXTINF:-1 tvg-id="TVGuara.br" status="error",TV Guará (720p) [Not 24/7]
+https://59f2354c05961.streamlock.net:1443/tvguara/_definst_/tvguara/chunklist.m3u8
```
Comment (LaneSh4d0w) [inline streams/br.m3u:243]:
```suggestion
https://video02.logicahost.com.br/tvguara23/tvguara23/playlist.m3u8
```

File: streams/br.m3u [hunk 7]
```diff
@@ -235,3 +235,27 @@ https://evpp.mm.uol.com.br/geob_band/bandnewstv/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" status="online",CBTV Now (1080p)
+https://oj7lng29dg82-hls-live.5centscdn.com/lives/f7b44cfafd5c52223d5498196c8a2e7b.sdp/index.m3u8
+#EXTINF:-1 tvg-id="" status="online",TV Brusque (720p)
+https://5ad482a77183d.streamlock.net/rodrigotvbrusque.com.br/_definst_/5d880199c902eb4a1e8df00d/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/br.m3u:247]:
```suggestion
https://5ad482a77183d.streamlock.net/rodrigotvbrusque.com.br/5d880199c902eb4a1e8df00d/playlist.m3u8
```
definst not needed

File: streams/br.m3u [hunk 8]
```diff
@@ -235,3 +235,27 @@ https://evpp.mm.uol.com.br/geob_band/bandnewstv/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" status="online",Mundo TV
+https://59f1cbe63db89.streamlock.net:1443/mundotv/_definst_/mundotv/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/br.m3u:249]:
```suggestion
https://59f1cbe63db89.streamlock.net:1443/mundotv/mundotv/playlist.m3u8
```
definst not needed

File: streams/br.m3u [hunk 9]
```diff
@@ -235,3 +235,27 @@ https://evpp.mm.uol.com.br/geob_band/bandnewstv/playlist.m3u8
...
+#EXTINF:-1 tvg-id="TVMAX.br" status="online",TV Max
+https://59f1cbe63db89.streamlock.net:1443/tvmax/_definst_/tvmax/playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/br.m3u:251]:
```suggestion
https://59f1cbe63db89.streamlock.net:1443/tvmax/tvmax/playlist.m3u8
```
definst not needed

File: streams/uk.m3u [hunk 10]
```diff
@@ -405,3 +405,9 @@ https://ve-uhd-push-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:uhd_stream_
 https://ve-uhd-push-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:uhd_stream_05/iptv_uhd_v1.mpd
 #EXTINF:-1 tvg-id="" status="error",Channel S (US Pacific) (720p) [Not 24/7]
 https://a.jsrdn.com/broadcast/22679_24MrQma9TX/-0800/c.m3u8
+#EXTINF:-1 tvg-id="" status="online",Shelanu TV (720p)
+https://1247634592.rsc.cdn77.org/1247634592/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="" status="online",Latest TV [Not 24/7]
+https://5a0e89631aa14.streamlock.net/LatestTV/LatestTV/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",La Muscle TV (360p)
```
Comment (UltraHDR) [inline streams/uk.m3u:412]: This is not live

File: streams/iq.m3u [hunk 11]
```diff
@@ -85,3 +85,13 @@ https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
 https://live.karwan.tv/karwan.tv/waar-tv/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="timeout",Music Al-Remas (720p) [Not 24/7]
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/c/musicalrmastv1/live
+#EXTINF:-1 tvg-id="DijlahTV.iq" status="online",Dijlah TV
+https://ghaasiflu.online/Dijlah/index.m3u8
+#EXTINF:-1 tvg-id="DijlahTarab.iq" status="online",Dijlah Tarab
+https://ghaasiflu.online/tarab/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="AlRasheedTV.iq" status="online",Al-Rasheed (408p)
+https://media1.livaat.com/AL-RASHEED-HD/tracks-v1a1/mono.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:93]:
```suggestion
https://media1.livaat.com/AL-RASHEED-HD/index.m3u8
```

File: streams/iq.m3u [hunk 12]
```diff
@@ -85,3 +85,13 @@ https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
...
+#EXTINF:-1 tvg-id="AlSharqiyaNews.iq" status="online",Al-Sharqiya News (1080p)
+https://5d94523502c2d.streamlock.net/alsharqiyalive/mystream/chunklist_w449457930.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:95]:
```suggestion
https://5d94523502c2d.streamlock.net/alsharqiyalive/mystream/playlist.m3u8
```

File: streams/iq.m3u [hunk 13]
```diff
@@ -85,3 +85,13 @@ https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
...
+#EXTINF:-1 tvg-id="AlSharqiya.iq" status="online",Al-Sharqiya (1080p)
+https://5d94523502c2d.streamlock.net/home/mystream/chunklist_w1408191520.m3u8
```
Comment (UltraHDR) [inline streams/iq.m3u:97]:
```suggestion
https://5d94523502c2d.streamlock.net/home/mystream/playlist.m3u8
```

File: streams/iq.m3u (hunk 11 above)
Comment (LaneSh4d0w) [reply streams/iq.m3u:93]: and quality is said to be 1080p, not 408p

File: streams/iq.m3u [hunk 14]
```diff
@@ -85,3 +85,13 @@ https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
 https://live.karwan.tv/karwan.tv/waar-tv/playlist.m3u8
 #EXTINF:-1 tvg-id="" status="timeout",Music Al-Remas (720p) [Not 24/7]
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/c/musicalrmastv1/live
+#EXTINF:-1 tvg-id="DijlahTV.iq" status="online",Dijlah TV
+https://ghaasiflu.online/Dijlah/index.m3u8
+#EXTINF:-1 tvg-id="DijlahTarab.iq" status="online",Dijlah Tarab
+https://ghaasiflu.online/tarab/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="AlRasheedTV.iq" status="online",Al-Rasheed (408p)
```
Comment (UltraHDR) [inline streams/iq.m3u:92]:
```suggestion
#EXTINF:-1 tvg-id="AlRasheedTV.iq" status="online",Al-Rasheed (1080p)
```

File: streams/iq.m3u (hunk 11 above)
Comment (UltraHDR) [reply streams/iq.m3u]: fixed

File: streams/br.m3u [hunk 15]
```diff
@@ -235,3 +235,27 @@ https://evpp.mm.uol.com.br/geob_band/bandnewstv/playlist.m3u8
...
+#EXTINF:-1 tvg-id="" status="online",Mundo TV
+https://59f1cbe63db89.streamlock.net:1443/mundotv/mundotv/playlist.m3u8
+#EXTINF:-1 tvg-id="TVMAX.br" status="online",TV Max
+https://59f1cbe63db89.streamlock.net:1443/tvmax/tvmax/playlist.m3u8
+#EXTINF:-1 tvg-id="TVUniaoFortaleza.br" status="online",TV União (720p) [Not 24/7]
+https://596639ebdd89b.streamlock.net/tvuniao/tvuniao/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",TV Marajoara (720p)
+https://video01.kshost.com.br:4443/tv31966/tv31966/playlist.m3u8
+#EXTINF:-1 tvg-id="TVBirigui.br" status="online",TV Birigui (640p)
+https://59f2354c05961.streamlock.net:1443/tvdigitalbirigui/_definst_/tvdigitalbirigui/chunklist_w763334596.m3u8
```
Comment (UltraHDR) [inline streams/br.m3u:257]:
```suggestion
https://59f2354c05961.streamlock.net:1443/tvdigitalbirigui/_definst_/tvdigitalbirigui/playlist.m3u8
```

File: streams/uk.m3u (hunk 10 above)
Comment (LaneSh4d0w) [reply streams/uk.m3u:412]: Understanding that the name is "LA Muscle TV", they have WAAAAAAAAAAAY too much video links inside their player. I think i'll need to deduct it from the link we have.

Comment (LaneSh4d0w) [reply streams/uk.m3u:412]: (and their "live TV" isn't really one. it's just a bunch of their vids put place in place to make it "look like it's live")

Comment (LaneSh4d0w) [reply streams/uk.m3u:412]: image

**Review by sguinetti — DISMISSED** [review]

**Review by sguinetti — DISMISSED** [review]

**Review by UltraHDR — APPROVED** [review]

**Review by sguinetti — APPROVED** [review]

**Pull request merged**